app/templates/index.html

```diff
@@ -13,14 +13,36 @@
 	                            <a href="static/badges/{{msg}}.badges.pdf" class="btn btn-success" download="download">Download as PDF</a>
 	                            <a href="static/badges/{{msg}}.badges.svg.zip" class="btn btn-success" download="download">Download as ZIP of SVGs</a>
 	                        </div>
+	                    {% endif %}
+
+	                    {% if cat == 'success-pdf' %}
+	                    	<div class="text-center">
+		                    	<div class="flash-{{cat}}">Your badges has been successfully generated.</div>
+		                    	<a href="static/badges/{{msg}}.badges.pdf" class="btn btn-success" download="download">Download as PDF</a>
+	                    	</div>
+	                    {% endif %}
+
+	                    {% if cat == 'success-zip' %}
+	                    	<div class="text-center">
+		                    	<div class="flash-{{cat}}">Your badges has been successfully generated.</div>
+		                    	<a href="static/badges/all-badges.zip" class="btn btn-success" download="download">Download as ZIP</a>
+		                    	<a href="static/badges/{{msg}}.badges.svg.zip" class="btn btn-success" download="download">Download as ZIP of SVGs</a>
+	                    	</div>
+	                    {% endif %}
+
                         {% else %}
                             <div class="flash-{{cat}}">{{msg}}</div>
-					    {% endif %}
 					{% endfor %}
 				{% endif %}
 			{% endwith %}
 			<form action="{{ url_for('upload') }}" method="post" enctype="multipart/form-data">
 			    <div class="form-group">
+			    	<label>Tick the download type(s):</label>
+			    	<label for="zip">Zip</label>
+			    	<input type="checkbox" id="zip" name="zip">
+			    	<label for="pdf">PDF</label>
+			    	<input type="checkbox" id="pdf" name="pdf">
+			    	<br>
```

Comment:
Please replace this `<br>` and use css margin to get space here.

Reply:
can you help me out with this one?

Reply:
Sure! 
Step 1: Remove "br" tag
Step 2: Wrap line 40 to 44 inside <span class="options">
Step 3: Add this to `style.css`
```
.options{
margin-bottom: 20px;
}
```

Reply:
i'm supposed to wrap it inside what ?

Reply:
@gabru-md Sorry, forgot to add "`"